ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/content/MonoGame.Application.WindowsDX.CSharp/MGNamespace.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>

```

Comment:
Didn't you say in the forums that Windows projects should target .NET Framework for smaller binaries?

Reply:
ILLink.Tasks nuget solves that problem quite nicely.

Reply:
Ooo that sounds useful! I'm assuming that still needs to be added here unless I missed it.

Reply:
Oops, this should target .NET 4.5 because that's what WindowsDX targets (because WinForms isn't available in stable .NET Core). And also the binary size will still be smaller than a self-contained build with .NET Core. But ILLink is really nice.